lib/ansible/module_utils/network/common/utils.py

```diff
@@ -442,6 +443,123 @@ def _fallback(fallback):
         pass
 
 
+def generate_dict(spec):
+    """
+    Generate dictionary which is in sync with argspec
+
+    :param spec: A dictionary which the argspec of module
```

Comment:
take this w/ a grain of salt, but I'm not sure what this description means.  Sounds like param_spec is the argspec of the module?  If so please change to  A dictionary that is the argspec of the module